timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -469,6 +470,7 @@ private static void assertNumberOfThreadsReasonable(int startingThreads, int thr
 
     private void abandonLeadershipPaxosModeAgnosticTestIfRunElsewhere() {
         Assume.assumeTrue(cluster == FIRST_CLUSTER);
+        Assume.assumeFalse(cluster.isDbTimelock()); // We will never test only DB timelock when releasing.
     }
```

Comment:
I did not understand the comment, could you please elaborate on that.

Reply:
There are some slow tests that aren't dependent on timestamp bound persistence (check where this is called), and so this skips them when running in a db timelock environment. This is only safe if we are sure it is tested elsewhere (which it is, because we will never test only db timelock when releasing).

Reply:
Makes sense